vertical-pod-autoscaler/hack/generate-crd-yaml.sh

```diff
@@ -32,8 +32,7 @@ trap cleanup EXIT
 if [[ -z $(which controller-gen) ]]; then
     (
         cd $WORKSPACE
-	      go mod init tmp
-	      go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0
+	      go install sigs.k8s.io/controller-tools/cmd/controller-gen@latest
```

Comment:
Can we get an explicit version here? Using latest is usually against best practices...

Reply:
Absolutely correct, thanks! Done with the newest commit.